CHANGELOG.rst

```diff
@@ -184,6 +184,10 @@ Bugfixes
 [1.9.6] - 2020-04-15
 ^^^^^^^^^^^^^^^^^^^^
 
+Features
+--------
+- `#5706 <https://github.com/RasaHQ/rasa/pull/5706>`_: Messenger Facebook Channel - Message typing in-off option + delay on message delivery.
+
```

Comment:
That should not go in here. Can you please create a file called `5706.feature.rst` with the above message under the directory `changelog`? Thanks.